### ISSUE TITLE
Package has unsupported layout for linux

### ISSUE THREAD

**Issue by rtvanals** [WeirdMath/SwiftyHaru#7]
# **Description**
Getting a the `package has an unsupported layout` error when `swift build`

```
error: the package has an unsupported layout, modulemap (*****/Project/.build/checkouts/SwiftyHaru.git--4030071447907157648/Sources/CLibPNG/module.modulemap) is not allowed to be mixed with sources
fix: move the modulemap inside include directory
```

# **Steps To Reproduce**

My `Package.swift`

```
import PackageDescription

let package = Package(
  name: "Utilities",
  dependencies: [
    .Package(url: "https://github.com/WeirdMath/SwiftyHaru.git",   majorVersion: 0, minor: 1)
  ]
)
```

In terminal, run command

`swift build -Xlinker -lz`

# Swift Version

Apple Swift version 3.1 (swiftlang-802.0.53 clang-802.0.42)
Target: x86_64-apple-macosx10.9

**Comment by broadwaylamb** [comment]
Hello @rtvanals! Thank you for your report.

It happens due to the new layout rules in Swift 3.1. In Swift 3.0 (the 0.1.0 release of SwiftyHaru targeted Swift 3.0) it worked just fine.

This problem has already been fixed in the `dev` branch. I'm going to release the 0.2.0 version in a couple of days. Until that you can just delete the `modulemap` files, it should work.

Please let me know if anything else goes wrong.

**Comment by broadwaylamb** [comment]
@rtvanals I've just published the [0.2.0 release](https://github.com/WeirdMath/SwiftyHaru/releases/tag/0.2.0). Can you try it and tell if everything is OK?

**Comment by rtvanals** [comment]
@broadwaylamb Everything seems fixed in the 0.2.0 release.  Thanks for the quick response!
